repos/fdbt-site/src/pages/products/multiOperatorProducts.tsx

```diff
@@ -0,0 +1,227 @@
+                                  <td className="govuk-table__cell dft-table-wrap-anywhere">
+                                      <a
+                                          href={`/products/productDetails?productId=${product.id}`}
+                                          id={`product-link-${index}`}
+                                      >
+                                          {product.productDescription}
+                                      </a>
+                                  </td>
+                                  <td className="govuk-table__cell">{sentenceCaseString(product.type)}</td>
+                                  <td className="govuk-table__cell">{product.duration}</td>
+                                  <td className="govuk-table__cell dft-table-wrap-anywhere">
+                                      {sentenceCaseString(product.passengerType)}
+                                  </td>
+                                  <td className="govuk-table__cell">{product.startDate}</td>
+                                  <td className="govuk-table__cell">{product.endDate}</td>
+                                  <td className="govuk-table__cell">
+                                      {getTag(product.startDate, product.endDate, true)}
+                                  </td>
+                                  <td className="govuk-table__cell">
+                                      <form>
+                                          <button
+                                              className="govuk-link govuk-body button-link"
+                                              formAction={buildCopyUrl(product.id.toString(), csrfToken)}
+                                              formMethod="post"
+                                              type="submit"
+                                              id="copy-product-button"
+                                          >
+                                              Copy
+                                          </button>
+                                      </form>
+                                  </td>
+                                  <td className="govuk-table__cell">
+                                      <button
+                                          className="govuk-link govuk-body button-link"
+                                          onClick={() => deleteActionHandler(product.id, product.productDescription)}
+                                      >
+                                          Delete
+                                      </button>
+                                  </td>
+                              </tr>
+                          ))
+                        : null}
+                </tbody>
+            </table>
+            {multiOperatorProducts.length === 0 ? (
+                <span className="govuk-body">
+                    <i>You currently have no multi-service products</i>
```

Comment:
multi-operator products, not multi-service

Reply:
corrected in other products too

Reply:
reverted [1104edc](https://github.com/fares-data-build-tool/create-data/pull/582/commits/1104edce2b6678c4769b3a464b723b67522f49be)